dashboard/app/controllers/levels_controller.rb

```diff
@@ -110,6 +110,46 @@ def index
     @levels = @levels.page(params[:page]).per(LEVELS_PER_PAGE)
   end
 
+  # GET /levels/get_filters/
+  # Get all the information to filter levels with
+  def get_filters
+    return head :no_content unless Rails.application.config.levelbuilder_mode
+
+    # Define search filter fields
+    @search_fields = {
```

Comment:
Are these (and the queries in `get_filtered_levels`) duplicated from `index`? What's the relationship between the two, and can/should we reuse it or extract common functionality?

Reply:
There are similarities between what is in index and what is the new methods but I also changes some things so I don't know if its worth extracting something. I think in the future we may want to have `/levels` use the new react components

Reply:
Thanks for the suggestion. Fixed this up